fetch.bs

```diff
@@ -3775,6 +3932,75 @@ steps:
  <!-- This is really bad and needs to be handled differently at some point. -->
 </ol>
 
+<p>To <dfn for=response>Attach timing info</dfn>, given <a for=/>response</a>
```

Comment:
Lowercase attach. Can probably leave out `for=response` here given it's an internal algorithm.

Reply:
Also, if we only do this once, I think it would be okay to inline this.